src/components/DatePicker/DatePicker.stories.mdx

```diff
@@ -164,9 +148,9 @@ Implementation of the regular DatePicker with options
   </Story>
 </Preview>
 
-# DatePicker with Filter
+### DatePicker with Filter
```

Comment:
do we have something on figma on this? is this still relevant?

Reply:
after discussion with the Design Team, we will leave this variation as it is for now. Some changes may occur in future tasks.